src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -520,6 +521,12 @@ Subgraph::ControlFlowPasses Subgraph::getControlFlowPasses() const {
                                     ov::snippets::lowered::pass::MarkLoops,
                                     ov::intel_cpu::pass::BrgemmCPUBlocking);
 
+#ifdef SNIPPETS_DEBUG_CAPS
+    SNIPPETS_REGISTER_PASS_RELATIVE(Place::After,
```

Comment:
I think we need to register this pass only if `linear_ir.debug_config.dumpParams` is not empty

Reply:
Got it, you are right. Can we check that inside the `run` at least?